### PR TITLE
adding API for pulling partials and partial metadata

### PR DESCRIPTION
**Pull request by gvn**

### CODE REVIEW

**Comment by gvn** [comment]
Resolves #40

File: README.md [hunk 1]
```diff
@@ -45,3 +45,16 @@ Templates for the Mozilla Foundation CMS.
...
+This project contains a very basic REST API which is deployed as static JSON routes to GitHub Pages.
+
+The service is available at:
+
+`http://mozilla.github.io/mozmaker-templates/api`
+
+### Routes
+
+- `GET /partials` - Returns an array of partial IDs.
+- `GET /partial/PARTIAL_ID` - Returns an object containing the complete markup for the indicated partial.
```
Comment (mmmavis) [inline README.md:60]: @gvn should we add an `partial_id` example here?

**Comment by mmmavis** [comment]
@gvn 
Wow nice hack and it looks great to me! :+1: 

One thing I'm not sure if it needs fix or not

Hitting http://127.0.0.1:1980/partials/3-boxes/ give me the following warning
screen shot 2016-03-21 at 10 33 42 am

I think the missing `</body>` warning is neglectable since we just returning a partial, not a full HTML doc. For the images I wonder if we should turn relative path into something on `https://mozilla.github.io/mozmaker-templates`. That way when we load template partial code into Calypso the placeholder images can show properly.

**Comment by gvn** [comment]
Yeah, that's livereload trying to inject itself into the partial, which isn't needed as its already injected into the wrapper. It's a development-only situation and safe to ignore.

Eventually we should put the images on a CDN, but you're right we could put absolute paths to the assets on GitHub Pages for now. Do you want to file a ticket for that?

**Pull request merged**